torntools/scripts/js/globalFunctions.js

```diff
@@ -1503,6 +1503,8 @@ const STORAGE = {
 				drug_details: true,
 				itemmarket_links: false,
 				highlight_bloodbags: "none",
+				show_missing_plushies: true,
```

Comment:
I think these should be `false` by default because most people won't need them and it might confuse certain users.

Reply:
Done.